app/views/admin/content/_merge.html.erb

```diff
@@ -0,0 +1,13 @@
+<div>
+  <h3>Merge Articles</h3>
+  <form action="/admin/content/merge/<%= @article.id %>" method="get">
+    <div class='clearfix'>
+      <label for="merge_with" style="width: 55px; margin-right: 5px;"><%= _("Article ID")%></label>
```

Comment:
Eek! Is there a class you can use to do this styling instead of styling directly in the label?